### DISCUSSION TITLE
Simple vs Full Objects

### DISCUSSION THREAD

**Discussion by tscotn** [JohnnyCrazy/SpotifyAPI-NET#861] [General]
I want to reduce the number of requests I make to the Spotify API in a playlist management app. I get several playlists at once for several users using the `Playlists.GetUsers()` method in the `PlaylistClient` and filter on count of Followers, but because this returns a List of `SimplePlaylist` objects rather than `FullPlaylist` objects, I then need to make a new API request per playlist to get the follower count. I pass in a `PlaylistGetUsersRequest` object to limit the fields it returns to just the ones I need.

From what I can tell, the Followers field is the only difference between a `SimplePlaylist` object and a `FullPlaylist` object, with a lot of common code between the Simple and Full structures in general. Has there been any discussion around making it possible to get Full objects from these paginated/bulk methods like `GetUsers`? Looks like Spotify documentation on simplified object models has been removed on their end.

Could be worthwhile to have Full and Simple objects inherit from some abstract common 
object and to make methods return generics, or building some other method set for explicitly getting Full objects. If there's some obvious solution I'm missing, I'd appreciate the help! Thanks.

**Comment by JohnnyCrazy** [comment]
Hey tscotn,

Sorry for the late respons. I removed `SimplePlaylist` and all methods now return `FullPlaylist`. Will be part of the next release!

**Reply by tscotn** [reply]
Thanks so much!
